src/core/server/saved_objects/migrations/integration_tests/batch_size_bytes.test.ts

```diff
@@ -159,7 +159,7 @@ function createRoot(options: { maxBatchSizeBytes?: number }) {
       },
     },
     {
-      oss: true,
+      oss: false,
```

Comment:
Some test datasets contains objects of types registered from xpack plugins, I had to enable xpack for those tests to avoid re-generating the dataset, as the tests were failing due to the behavior reintroduced in this PR.